server/src/permissions/index.ts

```diff
@@ -17,24 +18,28 @@ export const authorizationError = {
 // We want to skip authorization checks during integration tests
 const skipAuthorizations = ENV === 'test'
 
+const checkUser = (user: User | undefined): Error | undefined => {
+  if (!user) {
+    return new Error(
+      `${authenticationError.code}: ${authenticationError.message}`,
+    )
+  }
+  if (!user.email) {
```

Comment:
I don't understand when this happen ? how can a user.email be null if user is not ?

Reply:
This happens when the idp doesn't provide an email for the user

Reply:
right